### PR TITLE
Update Gift cards endpoint name and descriptions

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to update name and descriptions (GiftCard API and GiftCard Provider Protocol API) (EDU-9115).

### CODE REVIEW

File: VTEX - Giftcard API.json [hunk 1]
```diff
@@ -809,7 +809,7 @@
             "post": {
                 "tags": ["Transaction"],
                 "summary": "Cancel GiftCard Transaction",
-                "description": "Creates a cancellation in the transaction. Cancel a item reservation or create a refund.",
+                "description": "This request can be used to:\r\n\r\n1. Cancel a transaction for a specific gift card.\r\n\r\n2. Cancel a item reservation or create a refund.",
```
Comment (julia-rabello) [inline VTEX - Giftcard API.json:812]:
```suggestion
                "description": "This request can be used to:\r\n\r\n1. Cancel a transaction for a specific gift card.\r\n\r\n2. Cancel an item's reservation or create a refund.",
```

File: VTEX - Giftcard Provider Protocol.json [hunk 2]
```diff
@@ -287,7 +287,7 @@
 					"Gift Cards"
 				],
 				"summary": "Create GiftCard",
-				"description": "Creates a giftcard in a giftcard provider.",
+				"description": "Creates a gift card for a specific user in a gift card provider.\r\n\r\n## Warning\r\n\r\n- The field `profileId` should be filled with a value that identifies the client. It is the ID itself.\r\n- The field `expiringDate` should be filled in ISO 8601 format `YYYY-MM-DDThh:mm:ss.fff`.",
```
Comment (julia-rabello) [inline VTEX - Giftcard Provider Protocol.json:290]: Sugestão: formatar warning como callout com `>⚠️`

File: VTEX - Giftcard Provider Protocol.json [hunk 3]
```diff
@@ -944,8 +944,8 @@
 				"tags": [
 					"Transactions"
 				],
-				"summary": "Create GiftCard Transaction Cancellation",
-				"description": "Creates a cancellation transaction to a giftcard.",
+				"summary": "Cancel GiftCard Transaction",
+				"description": "This request can be used to:\r\n\r\n1. Cancel a transaction for a specific gift card in a gift card provider.\r\n\r\n2. Cancel a item reservation or create a refund.",
```
Comment (julia-rabello) [inline VTEX - Giftcard Provider Protocol.json:948]:
```suggestion
				"description": "This request can be used to:\r\n\r\n1. Cancel a transaction for a specific gift card in a gift card provider.\r\n\r\n2. Cancel an item's reservation or create a refund.",
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - Giftcard Provider Protocol.json (hunk 2 above)
Comment (ricardoaerobr) [reply VTEX - Giftcard Provider Protocol.json]: Removi esta informação, pois já consta nos campos a serem preenchidos.

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**